Library/Homebrew/utils/github/actions.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module GitHub
```

Comment:
Feels like this should be in a `Utils` module too based on the name?

Reply:
Yeah, but the existing `GitHub` module is not.

Reply:
I'd consider moving those files around?

Reply:
Which ones? Move `GitHub` out of `utils`?

Reply:
Yeh.

Reply:
I'm not sure it's worth moving at this moment. I think eventually all modules should follow the file structure and also all of them should be contained in the `Homebrew` namespace.

Reply:
Cool.